From: Lubomir Rintel <lkundrak@v3.sk>
Date: Mon, 14 Jan 2019 21:16:08 +0100
Subject: [PATCH] USB: EHCI: ehci-mv: add MODULE_DEVICE_TABLE
References: bnc#1012628
Patch-mainline: 4.20.6
Git-commit: 70d0ba4cf79a0e73485b22d955991c6f27257376

commit 70d0ba4cf79a0e73485b22d955991c6f27257376 upstream.

This fixes autoloading the module by the OF compatible string.

Fixes: 813e18b18a87 ("USB: EHCI: ehci-mv: add DT support")
Cc: stable@vger.kernel.org
Signed-off-by: Lubomir Rintel <lkundrak@v3.sk>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/usb/host/ehci-mv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/ehci-mv.c b/drivers/usb/host/ehci-mv.c
index f26109eafdbf..66ec1fdf9fe7 100644
--- a/drivers/usb/host/ehci-mv.c
+++ b/drivers/usb/host/ehci-mv.c
@@ -302,3 +302,4 @@ MODULE_AUTHOR("Chao Xie <chao.xie@marvell.com>");
 MODULE_AUTHOR("Neil Zhang <zhangwm@marvell.com>");
 MODULE_ALIAS("mv-ehci");
 MODULE_LICENSE("GPL");
+MODULE_DEVICE_TABLE(of, ehci_mv_dt_ids);
-- 
2.20.1

